xbmc/addons/kodi-addon-dev-kit/include/kodi/tools/Time.h

```diff
@@ -0,0 +1,90 @@
+#pragma once
+/*
+ *  Copyright (C) 2005-2019 Team Kodi
+ *  Copyright (C) 2011-2012 Pulse-Eight Limited.
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#if defined(TARGET_DARWIN)
+#include <mach/mach_time.h>
+#include <CoreVideo/CVHostTime.h>
+#elif defined(TARGET_WINDOWS)
+#include <Windows.h>
+#include <time.h>
+#else
+#include <time.h>
+#endif
+
+namespace kodi
+{
+  namespace time
+  {
+    //==========================================================================
+    /// @brief Function to get current time in milliseconds
+    ///
+    /// @return Current time in milliseconds as a double value
+    ///
+    ///
+    /// ------------------------------------------------------------------------
+    ///
+    /// **Example:**
+    /// ~~~~~~~~~~~~~{.cpp}
+    ///
+    /// #include <kodi/tools/Time.h>
+    ///
+    /// ...
+    /// double time = kodi::time::GetTimeMs();
+    /// ...
+    /// ~~~~~~~~~~~~~
+    ///
+    inline double GetTimeMs()
+    {
+    #if defined(TARGET_DARWIN)
+      return (double) (CVGetCurrentHostTime() / (double)(CVGetHostClockFrequency() * 0.001));
```

Comment:
C++ style casts

Reply:
I look to change them here and on other places, was taken like them from p8platform.

Can someone quikly test that on Mac a:
```cpp
return static_cast<double>(CVGetCurrentHostTime() / static_cast<double>(CVGetHostClockFrequency() * 0.001));
```
works correct.

Reply:
The code is updated and tested on Windows and Linux, only not Mac, think it works but not confirmed!